stl/inc/__msvc_bit_utils.hpp

```diff
@@ -140,9 +140,26 @@ _NODISCARD int _Checked_x86_x64_countl_zero(const _Ty _Val) noexcept {
 
 #if defined(_M_ARM) || defined(_M_ARM64)
 #ifdef __clang__ // TRANSITION, GH-1586
+#ifdef _M_ARM64
+_NODISCARD constexpr int _Clang_arm_arm64_countl_zero(const unsigned char _Val) {
+    // TODO: Revert back to __builtin_clzs once clang longer generates worse code than __builtin_clz
+    return __builtin_clz(_Val) - (_Unsigned_integer_digits<unsigned int> - _Unsigned_integer_digits<unsigned char>);
+}
+
+_NODISCARD constexpr int _Clang_arm_arm64_countl_zero(const unsigned short _Val) {
+    // TODO: Revert back to __builtin_clzs once clang longer generates worse code than __builtin_clz
+    return __builtin_clz(_Val) - (_Unsigned_integer_digits<unsigned int> - _Unsigned_integer_digits<unsigned short>);
+}
+#else

```

Comment:
We don't care about ARM32 performance, as long as it compiles and runs properly, so it isn't worth having a distinct codepath for it here.